packages/ckeditor5-alignment/docs/features/text-alignment.md

```diff
@@ -11,6 +11,16 @@ The {@link module:alignment/alignment~Alignment} feature enables support for tex
 
 {@snippet features/text-alignment}
 
+## Related features
+
+There are more CKEditor 5 features that can help you organize your content:
+* {@link features/basic-styles Basic font styles} &ndash; The essentials, like **bold**, *italic* and others.
+* {@link features/headings Headings} &ndash; Divide your content into sections.
+* {@link features/title Document title} &ndash; Clearly divide your content into a title and body.
+* {@ features/indent Block indentation} &ndash; Organize your contents into clearly visually divided blocks, indent crucial paragraphs etc.
```

Comment:
```suggestion
* {@link features/indent Block indentation} &ndash; Organize your contents into clearly visually divided blocks, indent crucial paragraphs etc.
```

Reply:
Also, this feature does not divide anything - it works on block level, so you have the blocks already divided. It just makes them visually stand apart more, maybe. But in general, focus on what you can use it for - to indent text blocks, visually highlight crucial information, communicate hierarchy.